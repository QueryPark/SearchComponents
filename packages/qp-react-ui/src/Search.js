import React from 'react'

const Component = () => (
  <p>qp component lib</p>
)

export default Component
