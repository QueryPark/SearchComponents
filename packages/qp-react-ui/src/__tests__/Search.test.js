import React from 'react'
import renderer from 'react-test-renderer'

import Search from '../Search'

test('initial snapshot test', () => {
  const component = renderer.create(
    <Search />
  )

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
