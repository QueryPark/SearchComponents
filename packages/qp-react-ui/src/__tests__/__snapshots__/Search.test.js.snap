// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`initial snapshot test 1`] = `
<Component>
  <p>
    qp component lib
  </p>
</Component>
`;
